chronokit/narx/narx.py

```diff
@@ -0,0 +1,98 @@
+import torch
+import torch.nn as nn
+from chronokit.base._models import Model
+from chronokit.preprocessing._dataloader import DataLoader
+
+
+class NARXModel(torch.nn.Module):
+
+    def __init__(self, n_features, hidden_size):
```

Comment:
We might need to allow usage of regularization like dropout layers, batch normalization etc... if the user desires